docs/source/markdown/options/mount.md

```diff
@@ -32,7 +32,14 @@ Current supported mount TYPEs are **bind**, **devpts**, **glob**, **image**, **t
 
 	      · dst, destination, target: mount destination spec.
 
-	      Paths matching globs, are mounted on the destination directory with the identical name inside the container.
+	      When source globs are specified without the destination directory
+              the files and directories are mounted with their complete path
+	      within the container. When the destination is specified, the
+	      files and directories matching the glob on the base file name
+	      on the destination directory. The option
```

Comment:
I think:
```suggestion
	      on the destination directory are mounted. The option
```

Reply:
What is is trying to say is if you have

/tmp/foo1 /tmp/foo2 and do

podman run --mount type=glob,source=/tmp/foo\*:destination=/dest fedora 

/tmp/foo1 will be mounted on /dest/foo1
/tmp/foo2 will be mounted on /dest/foo2

Without destination:

podman run --mount type=glob,source=/tmp/foo\* fedora 

/tmp/foo1 -> /tmp/foo1
/tmp/foo2 -> /tmp/foo2